### PR TITLE
Staker info

### LINKED ISSUE #35: Getter function for Staker Info.
# Rationale

This feature is required to easily get any staker's info on the terminal itself. It will help us view whether a staker accepts delegation or not, how much they have staked, their address, what commission they charge, when they last committed, etc.

# Implementation

A new command has to be introduced, `./razor getStakerInfo --stakerId <staker_id>`
This should call a `getStakerInfo` method, which would get the data from the razor contracts.
A nice table look would be desirable, using different colours for different parameters.

[This](https://github.com/razor-network/contracts/blob/6a6b116155cb0b2e889c25f6c8389efd91021943/contracts/lib/Structs.sol#L11) is the staker struct in the contract.

### PR DESCRIPTION
**Pull request by ashish10677**
# Description

A new command `stakerInfo` is added in this PR. This command will enable the user to fetch any stakers details.

Fixes #35 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Tests written
- [x] Tested on local hardhat

**Test Configuration**:
* Contracts version: v0.1.76
* Hardware: Macbook M1

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by SkandaBhat** [comment]
Is address required as a flag? We should be able to get staker info without this.

**Comment by ashish10677** [comment]
> Is address required as a flag? We should be able to get staker info without this.

Removed the address flag

File: README.md [hunk 1]
```diff
@@ -95,6 +95,17 @@ Example:
 $ ./razor stake --address 0x5a0b54d5dc17e0aadc383d2db43b0a0d3e029c4c --value 1000 --pow 10
 ```
 
+### Staker Info
+
+If you want to know the details of a staker, you can use stakerInfo command.
+```
+$ ./razor stakerInfo --address <your_address> --stakerId <staker_id_of_the_staker>
```
Comment (SkandaBhat) [inline README.md:102]: readme needs to be fixed. address is still used here

Comment (ashish10677) [reply README.md]: Updated

**Review by SkandaBhat — APPROVED** [review]
lgtm

**Pull request merged**